include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -1153,6 +1153,40 @@ class CommonOperatorLoader : public ProtobufLoader {
     return Error::success();
   }
 
+  Error loadLogicalOps(llvm::StringRef typeName, const OpType &op) {
+    std::string opName = loadOperatorName(op);
+    NodeValue xNV;
+    ASSIGN_VALUE_OR_RETURN_ERR(xNV, getNodeValueByName(op.input(0)));
+    NodeValue yNV;
+    ASSIGN_VALUE_OR_RETURN_ERR(yNV, getNodeValueByName(op.input(1)));
+    auto *LHS = xNV.getNode();
+    auto *RHS = yNV.getNode();
```

Comment:
This shouldn't be necessary, you should pass in `xNV` and `yNV` into all the `createNodeWithBroadcast` below.